kornia/contrib/kmeans.py

```diff
@@ -0,0 +1,155 @@
+# based on https://github.com/subhadarship/kmeans_pytorch
+
+import torch
+
+
+class KMeans:
+    def __init__(
+        self,
+        num_clusters: int,
+        cluster_centers: torch.Tensor,
```

Comment:
from kornia.core import Tensor

```suggestion
        cluster_centers: Tensor,
```

Reply:
same for other annotations